superbit_lensing/medsmaker/superbit/medsmaker_mocks.py

```diff
@@ -618,8 +772,8 @@ def make_object_info_struct(self, catalog=None):
         return obj_str
 
 
-    def run(self,outfile='mock_superbit.meds', clobber=True, source_selection=False,
-            select_stars=False):
+    def run(self,outfile='mock_superbit.meds', clobber=False, source_selection=False,
+            select_truth_stars=False,psf_mode='piff'):
```

Comment:
We should add `psf_mode` as an `argpase` arg in the script that calls this file

Reply:
Agreed. I was trying to minimize the changes to `pipe.py` but I think supplying a `psf_mode` argument is unavoidable.

Reply:
The only change required to `pipe.py` for this is to add `psf_mode` as either a required or optional arg to the correct class variable in the module. Not a problem